src/test/jtx/Env.h

```diff
@@ -292,6 +292,17 @@ class Env
         The command is examined and used to build
         the correct JSON as per the arguments.
     */
+    using rpcCallback = std::function<bool(Json::Value const&)>;
+    const rpcCallback useDefaultCallback{};
+    const rpcCallback expectInternalRPCError = [](Json::Value const&) {
```

Comment:
Can we rename this to `rejectNever` and add a comment explaining that it is meant to be used for tests that expect an internal error and do not want to trigger a retry?

Reply:
Yes. Fixed.